test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +66,12 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    private void assertIsAnyNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
+    }
+
+    private void assertNotAnyNull(Object... objects) {
+        assertFalse(Utils.isAnyNull(objects));
```

Comment:
It's good to follow the style used by the code nearby 👍
Having said that, it is doubtful if extracting these two methods add any value.
You could use the assertFalse and assertTrue inline as there are no conversions needed unlike in the elementsAreUnique method.